pkg/envoy/boilerplate.go

```diff
@@ -266,6 +267,9 @@ func makeCluster(host, ca, healthPath string, timeout time.Duration, addresses [
 		},
 		TlsContext:   tls,
 		HealthChecks: healthChecks,
+		OutlierDetection: &v2cluster.OutlierDetection{
+			MaxEjectionPercent: &types.UInt32Value{Value: outlierPercentage},
+		},
```

Comment:
This will turn on outlier detection for all clusters which may not be wanted in some scenarios, I think I'd prefer to have the default turn this off and enable it by specifying the percentage via the flag.